core/templates/dev/head/editor/EditorServices.js

```diff
@@ -46,8 +46,8 @@ oppia.factory('explorationData', [
     var explorationData = {
       explorationId: explorationId,
       // Returns a promise that supplies the data for the current exploration.
-      getData: function() {
-        if (explorationData.data) {
+      getData: function(ignoreCache=false) {
```

Comment:
Default params don't seem to be well-supported: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters.

Let's not use them.